designer/client/src/stylesheets/processes.styl

```diff
@@ -21,3 +21,9 @@
 .validation-labels
     display inline-grid
     max-width fit-content
+
+.line-cut
```

Comment:
we don't want new stylus things
we want to get rid of stylus

Reply:
done